README.md

```diff
@@ -282,6 +282,10 @@ The following settings and options are exposed to you. My default configuration
 
 `minimum_travel_distance` - Minimum distance of travel before processing a new scan
 
+`minimum_travel_heading` - Minimum changing in heading to justify an update.
+
+`check_minimum_travels_precisely` - Whether to always check if either *`minimum_travel_distance`* or *`minimum_travel_heading`* is satisfied. With the default value *`false`*, the behavior suits most cases where, for example, rotational odometry is poor.
```

Comment:
I feel like this needs to be named better. This includes the angular check not just distance. I think it would be better to call it something that better describes this fact

Reply:
You are right, maybe this would describe well
```yaml
minimum_motion_distance
minimum_motion_heading
check_minimum_motions_precisely
```
But that would require refactoring the old ones. A bit long but `check_minimum_travel_params_precisely` maybe?

I named it `check_minimum_travels_precisely` since it is the prefix of both `minimum_travel_distance` and `minimum_travel_heading`.